bindings/nodejs/src/lib.rs

```diff
@@ -2,23 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #![allow(clippy::needless_borrow)]
+// Increased limit required specifically for the send_message method
+#![recursion_limit = "130"]
```

Comment:
Any reason why it needs to be exactly this value? According to https://doc.rust-lang.org/reference/attributes/limits.html the default is 128, so only 2 less :thinking:

Reply:
yes, we were at 125 before, moving some code to a method to reuse upped it to 129 :/